lib/ansible/utils/display.py

```diff
@@ -55,6 +55,10 @@
 from ansible.utils.singleton import Singleton
 from ansible.utils.unsafe_proxy import wrap_var
 
+if t.TYPE_CHECKING:
+    # avoid circular import at runtime
+    from ansible.executor.task_queue_manager import FinalQueue
```

Comment:
Is there a better way?

Reply:
No, I think this is pretty standard practice, but it'd be a good idea to `from __future__ import annotations` so you don't have to quote `FinalQueue` everywhere.

Reply:
> Is there a better way?

One way is to have ABCs that would be used in typing so you could structure the imports in a unidirectional manner. But I suppose this might be a big change for this PR.